std/string.d

```diff
@@ -2867,10 +2867,10 @@ auto stripLeft(Range)(Range input)
         }
         else
         {
-            auto inputSave = input.save;
+            auto save = input.save;
             auto dc = decodeFront(input);
             if (!std.uni.isWhite(dc))
-                return inputSave;
+                return save;
         }
```

Comment:
This was changed because of a local import of `save` caused ambiguity.  Now that it's back at module scope, putting this back the way it was.